src/app/docs/icon/icon-docs.component.html

```diff
@@ -131,10 +131,10 @@
     ></app-icon-docs-icon-list>
 
     <sky-docs-demo-page-section
-      heading="Custom Icons (Beta)"
+      heading="Custom icons (Beta)"
     >
       <p>
-        In addition to the Font Awesome library, SKY UX also provides a set of custom icons.
+        SKY UX also provides a set of custom icons.
```

Comment:
I would vote to keep the original sentence since it communicates we offer both FontAwesome and custom icons.

Reply:
I can add it back if you feel strongly, but it seemed unnecessary to me since we call that out in the overview at the top of the page. And the `also` implies that the custom icons are in addition to something. Also, the corresponding intro for the section that precedes explains that Font Awesome icons are supported, but it doesn't call out that we also support custom icons. It seems to me that if we don't need to call out that custom icons there, we don't really need to call out Font Awesome icons here. But let me know what you think. If you still think we should keep this, I'll add it back in.

Reply:
I didn't realize we communicated FontAwesome higher up the page. This is fine.